README.md

```diff
@@ -1 +1,191 @@
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Ruby codes used to decode Morse Code
+<!-- > - Fetch details on details page
+> - Outputs numeric data values. -->
+  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy of the project, use this link:
+
+```sh
+cd decode-morse-code-message
+git@github.com:kingno3l/decode-morse-message.git
+```
+
+<!-- SETUP -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+To setup this project:
+
+```sh
+Download latest version at RubyInstaller for windows on local machine
+
+Download wsl -install -d Ubuntu for virtual experience
+```
+
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor
+3. Internet Connection
+4. Git
+
+<!-- INSTALL -->
+
+### Install
+
+Install this project with RubyInstaller on windows on local machine.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+irb
+```
+
+
+<!-- ### Run tests
+
+To test the project, execute the following command:
+
+```sh
+npm run test
+``` -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]() -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **King Immanuel**
+
+- GitHub: [@Kingno3l](https://github.com/kingno3l)
+- Twitter: [@Kingno3l](https://twitter.com/kingno3l_)
+- LinkedIn: [@Kingno3l](https://www.linkedin.com/in/kingno3l)
+
+👤 **Esmatullah Qadeemi**
+
+- GitHub: [@githubhandle](https://github.com/Esmatullah12)
+- Twitter: [@twitterhandle](https://twitter.com/Esmatullah173)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/esmatullah-qadeemi-b9951821b/)
+
+
+<!-- ## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Improve User Experience** -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTION -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!--SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project, please give it some starts ⭐️⭐️⭐️⭐️⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Special acknowledgement to  Microverse for creating this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.